packages/cli/src/lib/_Types.ts

```diff
@@ -1,2 +1,51 @@
-export type ProcessExit = (exitCode?: number) => void;
+import type { Client as StatesRedisClient } from '@iobroker/db-states-redis';
+import type { Client as ObjectsRedisClient } from '@iobroker/db-objects-redis';
+
+export type ProcessExit = (exitCode: number) => void;
 export type GetRepository = (repoName: string | undefined, params: Record<string, any>) => Promise<Record<string, any>>;
+export type CleanDatabase = (isDeleteDb: boolean) => any;
```

Comment:
I'm wondering if the type name should make clear that these are functions/callbacks. Maybe
```suggestion
export type ProcessExitCallback = (exitCode: number) => void;
export type GetRepositoryHandler = (repoName: string | undefined, params: Record<string, any>) => Promise<Record<string, any>>;
export type CleanDatabaseHandler = (isDeleteDb: boolean) => any;
```
or something similar?